### PR TITLE
Updating NoSQL root README.md

### LINKED ISSUE #21: Update NoSQL Documentation
Currently, the README md file for the entire camp does not contain all information needed for an open-source contributor to understand the purpose of the camp. Finish the [md file](https://github.com/bitprj/BitCamp/tree/master/NoSQL).

### PR DESCRIPTION
**Pull request by emsesc**
Adding content to root README.md file of the NoSQL Bitcamp. Includes a newly developed curriculum.

### CODE REVIEW

**Comment by emsesc** [comment]
@Land-dev do you have any gifs from your previous project? We can place them in the documentation

**Comment by lazyplatypus** [comment]
are we only adding week 1 of the content?

**Comment by emsesc** [comment]
@lazyplatypus Nope, working on the rest of the weeks right now. Just wanted to make the PR for feedback.

**Comment by lazyplatypus** [comment]
make sure if it is not ready for merging, make it a "Draft" pr
Screen Shot 2021-01-03 at 6 28 48 PM

**Comment by emsesc** [comment]
@FardeenB @Land-dev Finished up the root README.md file based off of current draft curriculum.

File: NoSQL/README.md [hunk 1]
```diff
@@ -5,50 +5,68 @@ Data is all around us and knowing how to use and manipulate databases is an incr
 ## The Goal 🥅
 **Who?** Built with the audience of college and high school students that have minimal to advanced coding experience in mind. Most do not have experience with NoSQL databases and deploying web applications.
 
-**What?** Concise and interactive Github Learning Lab that introduces and develops skills related to NoSQL databases with [MongoDB](https://azure.microsoft.com/en-us/services/functions/)
+**What?** Concise and interactive Github Learning Lab that introduces and develops skills related to NoSQL databases with [MongoDB](https://www.mongodb.com/)
+
+**How?** Students gain experience through building a complete web app with a frontend (HTML, CSS, JS, Atlas Realms) and backend (MongoDB, Azure Services, NodeJS). The application utilizes machine learning, an external API, and NoSQL databases to recommend products to users based off of a picture. Authentication is used to incorporate the database, which stores a history of recommendations with a corresponding user.
```
Comment (fbablu) [inline NoSQL/README.md]: "...databases to recommend products to users based on the contents in an uploaded image (objects and text within the picture)."

**Review by fbablu — COMMENTED** [review]
Looks good so far!

File: NoSQL/README.md [hunk 2]
```diff
@@ -5,50 +5,68 @@ Data is all around us and knowing how to use and manipulate databases is an incr
...
-* *The project:* ...
+* *Starting out:* Week 1 will ease students into the course by introducing them to MongoDB Compass and basic concepts related to NoSQL databases. They will also set up necessary tools (GitHub and Git Bash).
+* *The project:* Weeks 2 through 4 will focus on building the Product Recommendation Engine, walking through 3 basic steps: coding the backend Azure Function, deploying the frontend on Atlas Realms, and finally implementing the MongoDB database and authentication.
 
-**Where?** Tools used include Github, Git Bash, Atlas Realms, and MongoDB Databases.
+**Where?** Tools used include Github, Git Bash, Atlas Realms, Azure Services, and MongoDB (Compass) Databases.
 
 ### Project Description
-4-week course on using MongoDB, Atlas Realms, and Deploying a Webapp.
+4-week course on using MongoDB, Azure Services, Atlas Realms, and Deploying a Webapp.
```
Comment (Land-dev) [inline NoSQL/README.md:19]: we are not using Atlas Realms

**Comment by Land-dev** [comment]
Learning objectives should be changed to reflect what week 1 readme.md shows

File: NoSQL/README.md (hunk 2 above)
Comment (emsesc) [reply NoSQL/README.md:19]: @Land-dev should we have them deploy the web app somewhere or should we just use live server then?

**Pull request closed without merging**